### PR TITLE
Permission added in order to run EE9 specific tests

### LINKED ISSUE #17544: Did not receive response from server
An issue at client container test bucket

### PR DESCRIPTION
**Pull request by BerksanAtes**
fixes #17544

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#build

File: dev/com.ibm.ws.jaxws.clientcontainer_fat/fat/src/com/ibm/ws/jaxws/clientcontainer/fat/AppClientTest.java [hunk 1]
```diff
@@ -77,48 +77,48 @@ public static void setUp() throws Exception {
...
-
-    @Test
-    public void testServiceInjectionMemberLevel() throws Exception {
-        String testName = "ServiceInjectionMemberLevel";
-        List<String> response = Arrays.asList("Hello Response from ServiceInjectionMemberLevel");
-        List<String> packages = Arrays.asList("com.ibm.ws.jaxws.test.wsr.client", "com.ibm.ws.jaxws.test.wsr.server.stub",
-                                              "com.ibm.ws.jaxws.test.wsr.clientddmerge");
-        CommonApi.createClientWithArgs(testName, testName, response, packages, testName, "helloClient/src/" + testName, getServerInfo());
-    }
+    // @Test
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.clientcontainer_fat/fat/src/com/ibm/ws/jaxws/clientcontainer/fat/AppClientTest.java]: Why are these commented out?

**Review by neuwerk — CHANGES_REQUESTED** [review]

**Comment by BerksanAtes** [comment]
#build

**Review by neuwerk — APPROVED** [review]
Thanks for doing this!

**Comment by BerksanAtes** [comment]
#libbybot

**Pull request merged**